unknown type name int64_t (build regression between 2.0.13 and 2.0.17?)

I am running into an error when building version `2.0.17` from source. The error didn't occur with version `2.0.13`. I'm hesitant to call this a regression as it is far more likely that I did something wrong. Both builds ran inside the same container image, `golang:1.13.8-alpine3.11`. 

Thank you!

[Error:](https://circleci.com/gh/marshallford/podman/42)
```
Step 8/27 : RUN git clone --branch v$CONMON_VERSION https://github.com/containers/conmon $GOPATH/src/github.com/containers/conmon &&     cd $GOPATH/src/github.com/containers/conmon && make
 ---> Running in 7d8bf8ff93c2
Cloning into '/go/src/github.com/containers/conmon'...
mkdir -p bin
cc -std=c99 -Os -Wall -Wextra -Werror -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -DVERSION=\"2.0.17\" -DGIT_COMMIT=\""41877362fc4685d55e0473d2e4a1cbe5e1debee0"\" -o src/conmon.o -c src/conmon.c
cc -std=c99 -Os -Wall -Wextra -Werror -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -DVERSION=\"2.0.17\" -DGIT_COMMIT=\""41877362fc4685d55e0473d2e4a1cbe5e1debee0"\" -o src/cmsg.o -c src/cmsg.c
cc -std=c99 -Os -Wall -Wextra -Werror -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -DVERSION=\"2.0.17\" -DGIT_COMMIT=\""41877362fc4685d55e0473d2e4a1cbe5e1debee0"\" -o src/ctr_logging.o -c src/ctr_logging.c
cc -std=c99 -Os -Wall -Wextra -Werror -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -DVERSION=\"2.0.17\" -DGIT_COMMIT=\""41877362fc4685d55e0473d2e4a1cbe5e1debee0"\" -o src/utils.o -c src/utils.c
cc -std=c99 -Os -Wall -Wextra -Werror -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include  -DVERSION=\"2.0.17\" -DGIT_COMMIT=\""41877362fc4685d55e0473d2e4a1cbe5e1debee0"\" -o src/cli.o -c src/cli.c
In file included from src/cli.c:1:
src/cli.h:33:8: error: unknown type name 'int64_t'
   33 | extern int64_t opt_log_size_max;
      |        ^~~~~~~
In file included from src/cli.c:3:
src/ctr_logging.h:10:49: error: unknown type name 'int64_t'
   10 | void configure_log_drivers(gchar **log_drivers, int64_t log_size_max_, char *cuuid_, char *name_, char *tag);
      |                                                 ^~~~~~~
src/cli.c:40:1: error: unknown type name 'int64_t'
   40 | int64_t opt_log_size_max = -1;
      | ^~~~~~~
src/cli.c: In function 'process_cli':
src/cli.c:164:2: error: implicit declaration of function 'configure_log_drivers' [-Werror=implicit-function-declaration]
  164 |  configure_log_drivers(opt_log_path, opt_log_size_max, opt_cid, opt_name, opt_log_tag);
      |  ^~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make: *** [Makefile:71: src/cli.o] Error 1
The command '/bin/sh -c git clone --branch v$CONMON_VERSION https://github.com/containers/conmon $GOPATH/src/github.com/containers/conmon &&     cd $GOPATH/src/github.com/containers/conmon && make' returned a non-zero code: 2
```

--- a/src/cli.h
+++ b/src/cli.h
@@ -1,8 +1,8 @@
 #if !defined(CLI_H)
 #define CLI_H
 
-#include <glib.h> /* gboolean and GOptionEntry */
-#include <stdint.h>
+#include <glib.h>   /* gboolean and GOptionEntry */
+#include <stdint.h> /* int64_t */
 
 extern gboolean opt_version;
 extern gboolean opt_terminal;

--- a/src/cli.h
+++ b/src/cli.h
@@ -2,6 +2,7 @@
 #define CLI_H
 
 #include <glib.h> /* gboolean and GOptionEntry */
+#include <stdint.h>
 
 extern gboolean opt_version;
 extern gboolean opt_terminal;

--- a/src/ctr_stdio.h
+++ b/src/ctr_stdio.h
@@ -1,8 +1,8 @@
 #if !defined(CTR_STDIO_H)
 #define CTR_STDIO_H
 
-#include <glib.h> /* GIOCondition and gpointer */
-#include <stdint.h>
+#include <glib.h>   /* GIOCondition and gpointer */
+#include <stdint.h> /* int64_t */
 
 gboolean stdio_cb(int fd, GIOCondition condition, gpointer user_data);
 void drain_stdio();

--- a/src/ctr_stdio.h
+++ b/src/ctr_stdio.h
@@ -2,6 +2,7 @@
 #define CTR_STDIO_H
 
 #include <glib.h> /* GIOCondition and gpointer */
+#include <stdint.h>
 
 gboolean stdio_cb(int fd, GIOCondition condition, gpointer user_data);
 void drain_stdio();

